scripts/add_extra_components.py

```diff
@@ -232,7 +266,7 @@ def attach_hydrogen_pipelines(n, costs, config):
         from _helpers import mock_snakemake
 
         os.chdir(os.path.dirname(os.path.abspath(__file__)))
-        snakemake = mock_snakemake("add_extra_components", simpl="", clusters=10)
+        snakemake = mock_snakemake("add_extra_components", simpl="", clusters=6)
```

Comment:
What has been the reason on this change?

Reply:
Oh, I was testing locally. I didnt intend to push this.